plugins/catalog-react/src/components/EntityOwnerPicker/EntityOwnerPicker.tsx

```diff
@@ -221,7 +221,7 @@ export const EntityOwnerPicker = (props?: EntityOwnerPickerProps) => {
         }}
         name="owner-picker"
         onInputChange={(_e, inputValue) => {
-          setText(inputValue);
+          setText(inputValue.toLocaleLowerCase('en-US'));
```

Comment:
I believe we still want users to be able to input upper case. Ideally only the comparison is done in lower case. 🤔